### DISCUSSION TITLE
How to obtain the number of distance computations of the search process for a hybrid query

### DISCUSSION THREAD

**Discussion by Molly-Xu00** [milvus-io/milvus#33588] [Q&A and General discussion]
For a hybrid query, the results it returns to me are the objects satisfying the conditions. I still want to know the number the distance compuations during vector search for the hybrid query, which is also a good metric to measure the efficiency. Is there any way I can obtain that?

**Comment by yhmo** [comment]
Currently the original distance is not returned.
If you want to obtain the original distance, a workaround is: use the same parameters to call search() again.

**Reply by Molly-Xu00** [reply]
Thank you for your answer. But actually, what I want to get is how many times does the ANN search process compute the distance between the data vectors and the query vector. Is this available?

**Reply by yhmo** [reply]
Not sure what you want.
If your milvus is a cluster, you can deploy a [monitoring system](https://milvus.io/docs/monitor.md) to observe the latency metrics:
![Screenshot from 2024-06-04 15-46-15](https://github.com/milvus-io/milvus/assets/2282099/34f4cc2b-36b7-4a8d-9fa6-abf052ec5595)

**Reply by xiaofan-luan** [reply]
currently no.
but it's actually a good metric to evaluate
